src/reducers/dashboards.js

```diff
@@ -117,31 +125,32 @@ export const sGetById = (state, id) =>
  * @param {Object} state The current state
  * @returns {Array}
  */
-export const sGetItems = state => sGetFromState(state).items;
+export const sGetDashboardItems = state => sGetFromState(state).items;
 
 // selector level 2
 
 /**
- * Generic selector which returns filtered dashboards
+ * Generic selector which returns dashboards filtered by a prop value
+ * Uses === to check equality
  *
  * @function
  * @param {Object} state The current state
- * @param {Object} propName The name of the filter prop
- * @param {Object} value The value of the filter prop
+ * @param {String} propName The name of the filter prop
+ * @param {Any} value The value of the filter prop
  * @returns {Array}
  */
-const sGetDashboardsByProp = (state, propName, value) =>
-    Object.values(orObject(sGetById(state))).filter(
+const sGetDashboardsFilteredByPropValue = (state, propName, value) =>
+    Object.values(orObject(sGetAllDashboards(state))).filter(
```

Comment:
I don't think the orObject is necessary here since it is in sGetAllDashboards

Reply:
Correct